moped-editor/src/auth/user.js

```diff
@@ -100,10 +100,7 @@ export const initializeUserDBObject = (userObject) => {
       body: JSON.stringify({
         query: ACCOUNT_USER_PROFILE_GET_PLAIN,
         variables: {
-          userId:
-            config.env.APP_ENVIRONMENT === "local"
-              ? 1
-              : getDatabaseId(userObject),
```

Comment:
I think that, as long as we test with either the test admin or test editor accounts locally, then we shouldn't need to default to `user_id` 1 anymore. Same with the other change below that defaults to `user_id` when showing the `Profile` component.

Reply:
i'm so glad you addressed this! it only added to the confusion of what the heck is going on with logins 😅